tests/cpptests/test_cpp_index.cpp

```diff
@@ -1503,12 +1503,15 @@ TEST_F(IndexTest, testIndexFlags) {
 TEST_F(IndexTest, testDocTable) {
   char buf[16];
   DocTable dt = NewDocTable(10, 10);
+  // dt.memsize = cap * sizeof(DMDChain *)
```

Comment:
```suggestion
  // dt.memsize = sizeof(DocTable) + cap * sizeof(DMDChain)
```
No?

Reply:
OK. Change done, thanks.
Additionally, the tests were updated to calculate the expected value.